website/blog/2025-05-20-podman-desktop-core.md

```diff
@@ -0,0 +1,142 @@
+
+```Dockerfile
+FROM nginxinc/nginx-unprivileged
+
+COPY <<EOF /usr/share/nginx/html/index.html
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Simple Nginx Container</title>
+</head>
+<body>
+    <h1>Hello from my Podman Nginx Container!</h1>
+    <p>This content is being served by Nginx running on a Podman container.</p>
+</body>
+</html>
+EOF
+```
+
+Once your application and Containerfile are ready, Podman Desktop makes the build process straightforward:
+![Build image](img/podman-desktop-core-blog/build-image.png)
+
+1. Navigate to the "Images" section
+2. Select your Containerfile
+3. Provide a name for your image (e.g. webserver)
+4. Click "Build"
+
+After building your image, you can immediately run it with a single click, and your container will appear in the "Containers" list.
+
+![Start container](img/podman-desktop-core-blog/start-container.png)
+
+1. Find your container on the “Images” list
+2. Click the “Run Image ▶️” button
+3. Give your container a name we will call it: “webserver”
+
+### Container Management and Log Analysis
+
+Now that our webserver container is running we can inspect it in the UI
+![View container details](img/podman-desktop-core-blog/container-details.png)
+
+We can click the Open Browser button to see the webserver in our browser:
+![Webserver open in browser](img/podman-desktop-core-blog/nginx-in-container.png)
+
+Monitoring container logs is crucial for debugging and understanding application behavior. Let's use Podman Desktop to view the logs of the newly created container.
+
+1. Select your running container from the "Containers" list
+2. Click on the "Logs" tab to view the container's logs
+3. In the newest version of Podman Desktop you can also search in the logs\!
+
+![View container logs](img/podman-desktop-core-blog/container-logs.png)
+
+With that, we can continue the development process. We are able to rebuild our container when we have updated code. On top of that we can share our Containerfile with our team who will be able to reproduce the exact same environment as us to build and test their code.
+
+## Working with Kubernetes and OpenShift
+
+Now that we have a development setup up and running it’s time to get ready for production. In today's world it is natural for us to move to Kubernetes. Having a locally running cluster, we are able to iterate quickly but still have an environment that is as close to production as possible. This aids in a smoother migration in the future. Podman Desktop is here to help you test and execute that migration.
+
+With [kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/) we are able to have a locally running Kubernetes cluster in minutes. This will allow us to test our application in a Kubernetes environment. Kind comes installed together with Podman Desktop so you will be able to get started instantly.
```

Comment:
```suggestion
With [kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/), we are able to have a locally running Kubernetes cluster in minutes. This will allow us to test our application in a Kubernetes environment. Kind comes installed together with Podman Desktop so you will be able to get started instantly.
```